dev/com.ibm.ws.opentracing.1.x_fat/fat/src/com/ibm/ws/testing/opentracing/test/FATSuite.java

```diff
@@ -43,18 +33,7 @@
     MicroProfile14NoTracer.class
 })
 public class FATSuite implements FATOpentracingConstants {
-    
-    private static Set<String> openTracing11 = new HashSet<String>(Arrays.asList("mpOpenTracing-1.1", "usr:opentracingMock-1.1", "microProfile-1.4", "servlet-3.1", "jaxrs-2.0"));
-    private static Set<String> openTracing12 = new HashSet<String>(Arrays.asList("mpOpenTracing-1.2", "usr:opentracingMock-1.2", "microProfile-1.4", "servlet-3.1", "jaxrs-2.0"));
-    private static Set<String> openTracing13 = new HashSet<String>(Arrays.asList("mpOpenTracing-1.3", "usr:opentracingMock-1.3", "microProfile-2.1", "servlet-4.0", "jaxrs-2.1"));
-    
-    @ClassRule
-    public static RepeatTests r = RepeatTests.withoutModification()
-                    .andWith(new FeatureReplacementAction(openTracing11, openTracing12).withID("OT12")
-                             .forceAddFeatures(false))
-                    .andWith(new FeatureReplacementAction(openTracing12, openTracing13).withID("OT13")
-                             .forceAddFeatures(false));
```

Comment:
I want to take a closer look at this. The sets of features you're repeating look like sensible things to test, but they're quite different from the set of features described here.